inc/datahandlers/user.php

```diff
@@ -324,14 +324,78 @@ function verify_website()
 	 */
 	function verify_icq()
 	{
+		global $mybb;
+
 		$icq = &$this->data['icq'];
 
+		if($mybb->settings['allowicqfield'] == '' || !is_member($mybb->settings['allowicqfield'], getuser($this->data['uid'])))
```

Comment:
I think the first `$mybb->settings['allowicqfield'] == ''` check is superfluous.

Also, isn't it `get_user()`?

Reply:
That function name typo slipped me by as I helped @Eldenroot with this PR. Looks like we need to add testing to the "lessons", because that wouldn't have made it through. And, without investigating, you're probably right about the superfluous check. We just based the condition on the existing code which we moved from outside the data handler. We should have been more cautious about that code.